release/pipline/dags/istio_common_dag.py

```diff
@@ -0,0 +1,251 @@
+"""Airfow DAG and helpers used in one or more istio release pipline."""
+
+"""Copyright 2017 Istio Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import datetime
+import time
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.operators.python_operator import BranchPythonOperator
+from airflow.operators.python_operator import PythonOperator
+
+import environment_config
+from gcs_to_gcs import GoogleCloudStorageToGoogleCloudStorageOperator
+
+YESTERDAY = datetime.datetime.combine(
+    datetime.datetime.today() - datetime.timedelta(days=1),
+    datetime.datetime.min.time())
+
+default_args = {
+    'owner': 'laane',
+    'depends_on_past': False,
+    'start_date': YESTERDAY,
+    'email': ['laane@google.com'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': datetime.timedelta(minutes=5),
+}
+
+
+def MakeCommonDag(name='istio_daily_flow_test',
+                  schedule_interval='15 3 * * *',
```

Comment:
Since cron falls outside the context of python I think there should be a comment saying 'what' it is, but also an explanation about the 'why' (even if it's just to say it was arbitrary, or because rkpagadala said so).  It helps people to know what other factors they should consider (if any) if they have a reason to change to another time.  Based on prior comments I heard I just thought the builds were firing at midnight UTC.